includes/class-sensei-teacher.php

```diff
@@ -254,19 +254,20 @@ public function teacher_meta_box_content( $post ) {
 		$current_author = $post->post_author;
 
 		// get the users authorised to author courses
-		$users = $this->get_teachers_and_authors();
+		$users = $this->get_teachers_and_authors_with_fields( [ 'id', 'display_name' ] );
```

Comment:
Checking the [MySQL documentation](https://dev.mysql.com/doc/refman/8.0/en/identifier-case-sensitivity.html), it wouldn't be case-sensitive for any platform, but I think we could rename the cases where we're using `id` to `ID`, just to keep the same capitalization of the DB, and what they recommend in [the docs](https://developer.wordpress.org/reference/classes/wp_user_query/).

Reply:
Done ✅